types/index.d.ts

```diff
@@ -55,5 +55,12 @@ export interface DotenvConfigOutput {
  *
  */
 export function config(options?: DotenvConfigOptions): DotenvConfigOutput;
+
+/** dotenv library interface */
+export interface dotenv {
```

Comment:
Can we export out this interface with the casing `DotEnv`? It would then match the casing of the other interface [here](https://github.com/motdotla/dotenv/pull/450/files#diff-88bec6beae84369c0376b56b3bb88fe1L44).

Reply:
actually I think i am wrong about this let me download the code and get back to you.

Reply:
Ok pulled down, a saw the behavior of this. I think we should Pascal case the interfaces. I would love for the typing to work with. 

```
import dotenv from 'dotenv'

dotenv.foo() // throws on compile.
dotenv.config() // works 
```

I wish I was better at typescript and could add some code of how to do this, but sadly I am not quite sure how to get this to work.

Reply:
To do that, we can use [namespace](https://www.typescriptlang.org/docs/handbook/namespaces.html) to make it 👍